src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -49,6 +49,9 @@ if (
   store.checkoutConfiguration.paymentMethodsConfiguration.paywithgoogle.configuration[
```

Comment:
what's the use of having 'merchantIdentifier' in a separate const called `id`?

Reply:
for live Google Pay payments the ID is required